plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -179,7 +180,9 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
 
   @VisibleForTesting(otherwise = PRIVATE)
   internal fun performMapJump(cameraOptions: CameraOptions) {
-    if (lastCameraOptions == cameraOptions) {
+    // no need to call native setCamera if camera did not change or
+    // empty camera object has arrived (could be the case for gestures)
+    if (lastCameraOptions == cameraOptions || cameraOptions.isEmpty()) {
```

Comment:
is it somehow related to view annotations?

Reply:
no, will remove and create follow-up ticket